File: summation_of_primes/python/summation_of_primes.py
import math

max = 2000000

primes = [2]

for i in xrange(3,max,2):
	prime = True
	limit = math.ceil(math.sqrt(i)) # tip from SO
	for j in xrange(3, int(limit)+1, 2):
		if prime:
			if i % j == 0:
				prime = False
	if prime:
		primes.append(i)

sum = 0
for prime in primes:
	sum += prime;

print sum;

File: summation_of_primes/php/summation_of_primes.php
<?php

$max = 2000000;

$primes = Array(2);

for ($i=3; $i<$max; $i+=2) {
	$prime = true;
	$limit = ceil(sqrt($i)); // tip from SO
	for ($j=3; $j<=$limit && $prime == true; $j+=2) {
		if ($i % $j == 0) {
			$prime = false;
		}
	}
	if ($prime) {
		array_push($primes, $i);
	}
}

$sum = 0;
foreach ($primes as $prime) {
	$sum += $prime;
}

echo $sum;
